common/src/tx/trust.rs

```diff
@@ -132,9 +133,13 @@ impl Decodable for TrustEntry {
     }
 }
 
+#[derive(Getters)]
 pub struct AcceptedTrustUpdates {
+    #[get = "pub with_prefix"]
```

Comment:
We should be consistent. Either we use prefix for all or none

Reply:
Done 1ccfa54